src/constants/localizedConstants.ts

```diff
@@ -0,0 +1,275 @@
+import * as nls from 'vscode-nls';
+
+// Localizable Strings (localized at runtime)
```

Comment:
Should we document how to add new strings to this? E.g. do we need to update the enu xliff file at the same time? What process will we have for keeping the strings in a localizable state?